src/indicators/ipc.R

```diff
@@ -0,0 +1,238 @@
+library(Ripc)
+library(tidyverse)
+library(countrycode)
+
+output_dir <- file.path(
+  Sys.getenv("AA_DATA_DIR"),
+  "private",
+  "exploration",
+  "glb",
+  "global_monitoring",
+  "outputs"
+)
+
+#############
+#### IPC ####
+#############
+
+# get into country level form
+df_ipc <- ipc_download() %>%
+  mutate(
+    iso3 = countrycode(country, origin = "country.name", destination = "iso3c"),
+    phase_4pl_num = phase_4_num + phase_5_num,
+    phase_4pl_pct = phase_4_pct + phase_5_pct
+  ) %>%
+  group_by(
+    country,
+    iso3,
+    date_of_analysis,
```

Comment:
for consistency with the other variables and reducing underscores, consider renaming `analysis_date` or `report_date`